README.md

```diff
@@ -2,12 +2,16 @@
 
 This repo is the canonical source for Operators that appear in [OperatorHub.io](https://operatorhub.io).
 
+## Before contribution
+
+All Operators on [OperatorHub.io](https://operatorhub.io) need to function on vanilla Kubernetes. If you mandate a certain Kubernetes version you express this in the metadata (see below).
```

Comment:
nit: I think having the second sentence is unnecessary. There is a link in this Readme that points to OLM's CSV guide, and in there the "MinKubeVersion" is the field they would want. If we are mentioning a specific metadata field here, then we should be more direct about pointing it out.
On the other hand I think singling this field of the metadata over others shows that it's somehow more important than other fields, which isn't the case.
Also, should there be mention that these operators need to be deployable via OLM?